v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -51,7 +51,7 @@ def from_dataset_configs(
             dataset_ht_globals = hl.eval(dataset_ht.globals)
             paths[dataset] = dataset_ht_globals.path
             versions[dataset] = dataset_ht_globals.version
-            enums[dataset] = dataset_ht_globals.enums
+            enums[dataset] = dict(dataset_ht_globals.enums)
```

Comment:
This was a bug I didn't catch until now because my test cases were wrong, they were enums that were already dicts: `enums={'test_enum': ['C', 'D']}`